src/pages/details/index.js

```diff
@@ -0,0 +1,83 @@
+import { MdFavorite, MdFavoriteBorder } from 'react-icons/md';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { IoIosArrowBack } from 'react-icons/io';
+import { useEffect } from 'react';
+import { deleteFavourite, setFavourite } from '../../redux/mainSlice';
+import styles from '../index.module.css';
+import ClicableElement from '../../components/Clicable';
+
+const Details = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const { singleCoin, favourites } = useSelector((state) => state.main);
+
+  const {
+    description,
+    id,
+    name,
+    img,
+    links,
+    ChnagePercentage,
+    currentPrice,
+  } = singleCoin;
```

Comment:
- [ ] [OPTIONAL] Please update your deployed version as we have some slight differences, take for example the pages description is different from what I can see on the local version.

Local Version  |  Deployed Version
:-------------------------:|:-------------------------:

![Screenshot from 2022-06-04 21-06-20](https://user-images.githubusercontent.com/58692590/172024248-498c505c-be19-4405-90bc-d89b7c35a8c3.png) | ![Screenshot from 2022-06-04 21-07-12](https://user-images.githubusercontent.com/58692590/172024276-dfec3c3d-d393-4807-9530-e2cd228811be.png)